patches/about-dialog-build-fix.patch

```diff
@@ -0,0 +1,26 @@
+From 0f00ce1f84664c1898670168ab97e4e3c10a9ec5 Mon Sep 17 00:00:00 2001
```

Comment:
This was fixed upstream. You should be able to remove this patch now. https://github.com/tenacityteam/tenacity/pull/427

Reply:
Yup, I was just doing that.